content/news/posts/2024/PSDevCon 2024-PrestaShop-UIKit.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+title: PSDevCon 2024 - PUIK (PrestaShop UI Kit) - The new UI kit from PrestaShop
+subtitle: Explore a summary of the PUIK presentation delivered at the second PrestaShop Developer Conference
+date: 2024-12-12
+authors: [ PrestaShop ]
+icon: icon-leaf
+tags: [news, event]
+---
+
+During the second PrestaShop Developer Conference on November 6, 2024, Matthias Goudjil introduced PUIK (PrestaShop UI Kit). This blog post provides a written summary of his presentation. During the talk, participants gained a clearer understanding of PrestaShop's UI Kit, based on the new PrestaShop Design System. 
```

Comment:
Please link Matthias name to his profile @mattgoud

Reply:
Hi Krystian, thanks for the comment - I've just added a link on the post to Matthias' profile on GitHub.